dev015-making-apps-more-powerful/04-custom-data-final/add-custom-data/Program.cs

```diff
@@ -42,28 +43,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            MyInformation myInformation = new MyInformation();
             try
             {
-                myInformation.DisplayMeAndMyManagerAsync().Wait();
+                RunAsync(args).GetAwaiter().GetResult();
```

Comment:
Refactoring: I moved all the demos to RunAsync, this way the error processing applies to all as well.